### PR TITLE
support customized schedule interval

### LINKED ISSUE #471: Enable recurring schedule of Notebooks and Pipelines
Looks like Kubeflow has enabled recurring schedule of experiments, and we could use to enable notebook and pipelines to be scheduled 

image

and the detailed configuration

image

### PR DESCRIPTION
**Pull request by wenjun93**
dag schedule interval is set '@once' by default, we should expose it as an user-defined option

Fixes #471

### CODE REVIEW

File: elyra/pipeline/airflow/processor_airflow.py [hunk 1]
```diff
@@ -400,13 +400,17 @@ def create_pipeline_file(self, pipeline, pipeline_export_format, pipeline_export
             if pipeline_description is None:
                 pipeline_description = f"Created with Elyra {__version__} pipeline editor using `{pipeline.source}`."
 
+            schedule_interval = pipeline.schedule_interval
```
Comment (lresende) [inline elyra/pipeline/airflow/processor_airflow.py:403]: Would it be better to have this in pipeline definition or pipeline value object directly?

File: packages/pipeline-editor/src/pipelineProperties.ts [hunk 2]
```diff
@@ -36,6 +42,19 @@ const pipelineProperties = {
         parameter_ref: 'runtime',
         label: { default: 'Pipeline Runtime' }
       },
+      {
+        control: 'custom',
+        custom_control_id: 'StringControl',
+        parameter_ref: 'schedule_interval',
+        label: {
+          default:
```
Comment (lresende) [inline packages/pipeline-editor/src/pipelineProperties.ts:50]: Could you work with @marthacryan or @ajbozarth and see if they could help get a better user experience.

**Comment by lresende** [comment]
Overall LGTM, but I would suggest collaborating with others on the community to come up with a better UX.

File: packages/pipeline-editor/src/pipelineProperties.ts (hunk 2 above)
Comment (wenjun93) [reply packages/pipeline-editor/src/pipelineProperties.ts:50]: @marthacryan @ajbozarth could please take a look and see how to improve the UX

**Comment by ptitzler** [comment]
Perhaps a better/cleaner approach would be to introduce this as a global pipeline property that's specific to Apache Airflow pipelines.

File: packages/pipeline-editor/src/pipelineProperties.ts [hunk 3]
```diff
@@ -36,6 +42,19 @@ const pipelineProperties = {
...
+            'Pipeline Schedule Interval(CRON expression, once if left empty)'
+        },
+        data: {
+          placeholder: '0 0 * * *',
```
Comment (akchinSTC) [inline packages/pipeline-editor/src/pipelineProperties.ts:54]: If possible (albeit more work), maybe we can make this more intuitive to use by include the airflow presets as an option via dropdown in addition to supporting the cron job notation e.g. left column here:
image

**Comment by akchinSTC** [comment]
See related if work scope is to expand : https://github.com/elyra-ai/elyra/issues/471

**Comment by lresende** [comment]
I have updated #471 with some more details on the requirements for this. It would be good if we could handle both Airflow and KFP runtimes for this feature, but if you are not completely familiar with KFP I can help. Also, if you need help on the ui field, please let me know and I will push a commit here but should be simple with the options from the airflow pattern.

File: packages/pipeline-editor/src/pipelineProperties.ts (hunk 2 above)
Comment (ajbozarth) [reply packages/pipeline-editor/src/pipelineProperties.ts:50]: sorry I didn't comment earlier, I actually though this through last week. I agree with Alan's comment below about leveraging the airflow presets. If we do that then this property can be an enum using the preset values and then the backend can interpret those values into the correct string for the runtimes that dont support the notation

**Comment by ptitzler** [comment]
Closing PR due to lack of activity. Please re-open if you still intend to work in this.

**Pull request closed without merging**